README.md

```diff
@@ -1,4 +1,24 @@
 # react-flowplayer
-Flowplayer React Component
+![React version](https://img.shields.io/badge/react->=17.0.2-green?style=flat)
+[![NPM version](https://img.shields.io/npm/v/@flowplayer/react-flowplayer.svg?style=flat)](https://npmjs.com/package/@flowplayer/react-flowplayer) 
+[![NPM downloads](https://img.shields.io/npm/dm/@flowplayer/react-flowplayer.svg?style=flat)](https://npmjs.com/package/@flowplayer/react-flowplayer)
+
+<br>
+<div>
+<a href="https://www.wowza.com">
+  <img src="./logo.svg" width="640">
+</a>
+</div>
+<br>
+
+**react-flowplayer** is the preferred way to use Flowplayer with the React framework. 
+
+### Resources
+<!-- [Documentation](https://developer.wowza.com/docs/wowza-flowplayer/guides/add-the-player-using-typescript.md/) -->
+* [Examples and Guides](https://developer.wowza.com/docs/wowza-flowplayer/player/embed/embed-with-standalone/#use-npm)
```

Comment:
@rj-wowza @ondreian For the React docs, maybe we should link to this instead? https://developer.wowza.com/docs/wowza-flowplayer/player/components/react/

It's definitely not wonderful, but all we got at the moment.